tools/aequatus/datatypes/aequatus_datatypes/datatypes_conf.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<datatypes>
+  <registration converters_path="lib/galaxy/datatypes/converters" display_path="display_applications">
+    <datatype extension="aequatus" type="galaxy.datatypes.text:Aequatus" mimetype="application/json" display_in_upload="true"/>
```

Comment:
Ok understandable, but this seems to be a Galaxy bug for me. Maybe worth to report?